.github/workflows/doxygen_errors.yml

```diff
@@ -0,0 +1,56 @@
+name: Doxygen Documentation Check
+
+on:
+  push:
+    branches: [ "master", "rdkcentral:master" ]
+  workflow_dispatch:
+
+jobs:
+  build-doxygen:
+      runs-on: ubuntu-20.04 
+      
+      steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install Doxygen
+        run: |
+          sudo apt-get update
+          sudo apt-get install doxygen
+      
+      - name: Install Graphviz
+        run: |
+          sudo apt install graphviz
+
+      - name: Make a doxygen_error.txt file
+        run: |
+          touch doxygen_errors.txt
+
+      - name: Run through the Doxygen Documentation
+        id: doxygen
+        run: |
+          mkdir -p build/gh_pages
+          doxygen 2> doxygen_errors.txt
```

Comment:
You don't need to redirect stderr when WARN_LOGFILE is set in Doxyfile. Doxygen will do it internally.